core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -115,6 +116,12 @@ oppia.directive('schemaBasedUnicodeEditor', [
             return $sce.trustAsHtml(
               $filter('convertUnicodeWithParamsToHtml')($scope.localValue));
           };
+
+          $scope.displayPopUpMessage = function() {
+            $timeout(function() {
+              return "Tap here to respond!";
```

Comment:
The return value here is not really used for anything. You'll want to define a $scope.textInputHelperMessage outside this timeout and initialize it to ''. Then here you would do `$scope.textInputHelperMessage = ...`.

That said, you really want a boolean here since the message is always the same. So maybe do the above with $scope.helperMessageIsShown = false; (or true;) and put an ng-if in the span.